AudibleApi/HttpClientSharer.cs

```diff
@@ -26,15 +27,14 @@ public HttpClientSharer()
 		}
 		public HttpClientSharer(HttpMessageHandler sharedMessageHandler)
```

Comment:
If you ever feel like replacing this with httpclientfactory, go for it. When I made it I was very annoyed that the IDisposible httpclient wasn't supposed to be disposed yet it kept state like the base url which made it hard to reuse. I didn't know about httpclientfactory at the time and it's possible it didn't yet exist. I haven't changed yet because I'm afraid of breaking the world and my orig solution still work, be it ever so clunky.

Reply:
Good to know for next time. I'll leave it for now though.